apps/firelens-stability/collections/ecs-firelens-stability-tests/s3-stability/cases/s3-throughput-30mbps.json

```diff
@@ -0,0 +1,5 @@
+{
+    "definitions": {
+        "datajet_kbps_throughput_tcp": "30000"
```

Comment:
future: split throughput through multiple streams of input, more like golden path

Reply:
real world high throughput use cases are probably k8s where throughput is spread over multiple tags from many pods

Reply:
Split it to 4 different tcps